bsip-0083.md

```diff
@@ -0,0 +1,80 @@
+
+# Rationale
+Because the BitAssets are currently controlled by platform-wide governance mechanisms, and because these are not well suited to the management of smart coins, which may at times need to respond quickly to market conditions, the authors propose to reorganize these assets to utilize a dedicated stakeholder-voted multisignature authority and thereby enable these assets to update in synchronization with market ebbs and flows.
+
+The block producers will no longer provide feeds for BitAssets.
+
+# Specification
+The following technical changes are required:
+
+**Assignment of Accumulated Fees**
+
+At the hardfork date, the accumulated fees of the existing BitAssets will be transferred to the committee-account.
+
+**Creation of BITASSET.MANAGER asset**
+
+At the hardfork date, a new BITASSET.MANAGER asset will be created with the committee account as its owner and all flags and permissions disabled. The initial distribution will be made equal to the current BTS distribution. BTS that are locked up in open orders, in collateral positions, in HTLC objects or in vesting balances will be assigned to the owners of the respective positions. Other BTS positions are out of scope (e. g. stealth).
+Subsequently, tokens of this new asset can be freely transferred or traded.
+
+**Add a feeding_authority extension to BitAssets**
+
+An extension will be added to `asset_update_bitasset_operation` that allows specifying an elected authority (see BSIP-0084) for determining the list of feed producers. `asset_publish_feed_operation` will be adapted to check that authority if the extension is present on a BitAsset.
+
+**Addition of voting process through BITASSET.MANAGER**
+
+Two new elected authorities (see BSIP-0084) will be created based on the BITASSET.MANAGER asset with settings that match the current voting mechanism (regarding min/max_number and proxy_allowed). The first one is for controlling the new bitasset-manager account (see below), the second one for selecting price feeders.
+
+**Update to Committee-Controlled BitAsset Issuer Fields**
+
+At the hardfork date, a new bitasset-manager account will be created with its active authority set to the first of the new elected authorities, and with the owner authority controlled by the committee.
+The committee-controlled BitAsset objects' issuer fields will be updated to specify this new account.
+After the hardfork, the new account will be able to assign other feed producers at its discretion.
+
+**Disable the witness_fed_asset and committee_fed_asset Flags**
+
+At the hardfork date, the `witness_fed_asset` and `committee_fed_asset` flags will be unset from the committee-owned BitAssets, and will be disabled from use in future assets. The existing price feeds will be cleared. The BitAssets will have the feeding_authority extension set to the second of the new elected authorities.
```

Comment:
It's dangerous to clear existing price feeds. It may lead to unexpected consequences.

Reply:
Without a valid feed neither forced settlement nor borrowing will be possible, until the new feeders take up their duties. This I think will happen quickly, because it is known in advance, and interested parties have already proven that they can act quickly when they see the need.

Thanks to the threshold price I wouldn't expect any forced settlements, and stopping the borrowing briefly shouldn't have any negative consequences.

Anything else?

Reply:
It doesn't sound like you're serious.

It's possible that at the hf time the price is above the threshold, or the threshold BSIP has been voted out.

Reply:
Even then, a brief interruption with ample pre-warning time is unlikely to cause much damage.

Reply:
Changed text for a smoother transition:
* no removal of feeders, only of flags
* instruct committee to enable new elected entities after voting has kicked in